### PR TITLE
[15.0][MIG] sale_report_margin: Migration to version 15.0

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT36541

please @LudLaf @cesar-tecnativa review this

### CODE REVIEW

**Review by LudLaf — APPROVED** [review]

File: sale_report_margin/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+* `Tecnativa <https://www.tecnativa.com>`_:
+
+    * Sergio Teruel <sergio.teruel@tecnativa.com>
+    * David Vidal <david.vidal@tecnativa.com>
```
Comment (victoralmau) [inline sale_report_margin/readme/CONTRIBUTORS.rst]:
```suggestion
    * Sergio Teruel
    * David Vidal
```

**Review by victoralmau — COMMENTED** [review]
Code review and functionality OK

File: sale_report_margin/i18n/sale_report_margin.pot [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: sale_report_margin
+#: model:ir.model.fields,field_description:sale_report_margin.field_sale_report__purchase_price
+msgid "Purchase Price"
+msgstr ""
```
Comment (LudLaf) [inline sale_report_margin/i18n/sale_report_margin.pot:19]:
```suggestion
msgstr "Coste"
```

File: sale_report_margin/i18n/sale_report_margin.pot [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+#. module: sale_report_margin
+#: model:ir.model,name:sale_report_margin.model_sale_report
+msgid "Sales Analysis Report"
+msgstr ""
```
Comment (LudLaf) [inline sale_report_margin/i18n/sale_report_margin.pot:24]:
```suggestion
msgstr "Análisis de ventas"
```

**Comment by LudLaf** [comment]
Functionality working as expected.

**Comment by CarlosRoca13** [comment]
@LudLaf The changes that you suggest are not correct. The translations are never set at file with extension `.pot` they are set at each specific language file with extension `.po`. For example, the Spanish translations are set at `es.po`.

@victoralmau change done

**Review by victoralmau — APPROVED** [review]

**Comment by LudLaf** [comment]
> @LudLaf The changes that you suggest are not correct. The translations are never set at file with extension `.pot` they are set at each specific language file with extension `.po`. For example, the Spanish translations are set at `es.po`.
> 
> @victoralmau change done

@CarlosRoca13 , noted. Thanks

**Pull request merged**